### PR TITLE
add new region eu-west-1 and use dedicated AMI for loader/monitor

### PR DESCRIPTION
**Pull request by amoskong**
We had added 4 SCT builders in eu-west-1 (Ireland) region, this PR added the new region in configure files. We also prepared two AMIs for loader (contains Shlomi's fixed c-s) and monitor.

### CODE REVIEW

**Comment by amoskong** [comment]
I just created a wiki page to maintain the loader/monitor AMI info:
- https://github.com/scylladb/scylla-cluster-tests/wiki/SCT-AMI-maintenance-page

We need to keep this page update.

**Comment by roydahan** [comment]
Good, please send to all QA in mail.

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2048,6 +2048,9 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
             self.kill_stress_thread()
             return
 
+        if self.params.get('skip_loader_setup', default=False):
```
Comment (bentsi) [inline sdcm/cluster.py]: What is this new parameter used for? And why for loader only?

Comment (roydahan) [reply sdcm/cluster.py]: I hate to add more yaml parameters.
I thought maybe we can set an environment variable or file on the AMI and during setup we can check it. if it's there, we can skip setup.

Comment (amoskong) [reply sdcm/cluster.py]: @bentsi If we use prepared loader AMI (contains everything: cs, collected configuration, scylla-bench, git/screen/etc), then we can skip the loader setup here. For monitor AMI, we still need to prometheus/grafana config, so we won't skip the setup as the loader.

@roydahan 
I agree with you, it's bad to introduce too many parameters to effect the logic.
At the beginning, I thought to identify the prepared AMI automatically. If c-s exists or a flag file exists (~/PREPARED-LOADER), then we will skip the loader setup, is it good?

Comment (roydahan) [reply sdcm/cluster.py]: Yes, it's good.
I prefer a flag file.

Comment (bentsi) [reply sdcm/cluster.py]: flag file in the AMi is a good idea  :+1:

Comment (amoskong) [reply sdcm/cluster.py]: I just updated the loader AMI (v2) & this PR.

Tested by http://jenkins.cloudius-systems.com:8080/job/scylla-staging/job/scylla-longevity-test-dev/label=aws-eu-west1-qa-builder2/ws/latest/job.log

**Comment by bentsi** [comment]
@amoskong are we ready to merge this?

**Comment by amoskong** [comment]
It's ready in my view. Let's get a final review/confirm with @roydahan

**Comment by roydahan** [comment]
I'll do a final review today and we will start migrating the jobs this week.

**Comment by roydahan** [comment]
I'm ok to merge.

**Pull request merged**